### PR TITLE
[Merged by Bors] - ET-3355 store last user id usage

### PR DESCRIPTION
**Pull request by rustonaut**
**References:**

- [ET-3355]

[ET-3355]: https://xainag.atlassian.net/browse/ET-3355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by biernacki — APPROVED** [review]
lgtm

File: justfile [hunk 1]
```diff
@@ -403,27 +403,33 @@ db-migrate +ARGS:
 web-ingestion-up: build-ingestion-service
     #!/usr/bin/env bash
     set -eux -o pipefail
+    compose="$(command -v podman-compose || command -v docker-compose)"
     rm -rf "$RUST_WORKSPACE/web-api/assets"
     mkdir -p "$RUST_WORKSPACE/web-api/assets"
-    ln -s "../../../$FLUTTER_WORKSPACE/example/assets/smbert_v0001/smbert.onnx" "$RUST_WORKSPACE/web-api/assets/model.onnx"
+    ln -s "../../../$FLUTTER_WORKSPACE/example/assets/smbert_v0001/smbert-quantized.onnx" "$RUST_WORKSPACE/web-api/assets/model.onnx"
```
Comment (acrrd) [inline justfile:409]: Not sure why these changes are here but for sure we don't want to start using the quantized model here.

**Review by acrrd — CHANGES_REQUESTED** [review]

File: justfile (hunk 1 above)
Comment (rustonaut) [reply justfile:409]: Because there is no `smbert.onnx` file there is just the mock or the quantize model.

Comment (rustonaut) [reply justfile:409]: Ok, we did change the smbert archive without bumping the version number so the data wasn't downloaded correctly.

**Review by biernacki — APPROVED** [review]

**Review by acrrd — APPROVED** [review]

**Comment by rustonaut** [comment]
bors merge
